Spatial.gd

```diff
@@ -284,27 +285,34 @@ func reset_minimap_masks():
 		minimap_path.material.set_shader_param("minimap_corner", compass_corner1)
 		minimap_path.material.set_shader_param("minimap_corner2", compass_corner2)
 
-var markerdata = Waypoint.Waypoint.new()
-var marker_file_dir = "user://protobins/"
+var Waypoint_data = Waypoint.Waypoint.new()
+var marker_file_dir = Directory.new()
```

Comment:
Now that this is being initialized in the _ready() function we should declare it before the _ready() function or figure out some way to set this value here with an onready like trigger